docs/blog/october-contributions-showcase.md

```diff
@@ -0,0 +1,165 @@
+      content: website
+  - - meta
+    - property: og:title
+      content: October contributions showcase
+  - - meta
+    - property: og:url
+      content: https://e18e.dev/blog/october-contributions-showcase
+  - - meta
+    - property: og:description
+      content: A showcase of e18e-focused contributions from October
+---
+
+# October contributions
+
+_October 4, 2024_
+
+A few days late again because I was caught in another typhoon :grimacing:, but here's the latest wins from the e18e community!
+
+## Libraries
+
+### picospinner & nanospinner
+
+Two small but great additions, these CLI spinner libraries are here to replace heavier weight alternatives like `ora`.
+
+[`picospinner`](https://github.com/tinylibs/picospinner/) was recently adopted into the [tinylibs](https://github.com/tinylibs) organisation, and is being lead by [Pondwader](https://github.com/PondWader).
+
+[`nanospinner`](https://github.com/usmanyunusov/nanospinner) author [@usmanyunusov](https://x.com/usmanyunusov) has also recently joined the e18e community and is actively helping move things forward.
+
+Both of these libraries are great alternatives and unblock a whole bunch of future migrations. Looking forward to see more!
+
+### empathic / fd-package-json
+
+As part of the cleanup project, we needed an alternative package for finding the closest `package.json`. To fill the gap, [`fd-package-json`](https://github.com/es-tooling/fd-package-json) was created **and recently passed 800,000 downloads/week!**
+
+However, we still had many gaps where we wanted to traverse for other files and directories without having to pull in a heavy library.
+
+This is where [empathic](https://github.com/lukeed/empathic) comes in!
+
+Another great library by [@lukeed](https://x.com/lukeed05), this provides a bunch of useful utilities for dealing with paths and traversals.
+
+This library can easily replace many of the `find-up`-like packages floating around the ecosystem:
+
+```ts
+import * as find from 'empathic/find'
+
+// Find closest "foobar.config.js" file
+const file = find.up('foobar.config.js', { cwd })
+```
+
+Let the migrations begin!
+
+### nano-staged
+
+Another one by [@usmanyunusov](https://x.com/usmanyunusov), [nano-staged](https://github.com/usmanyunusov/nano-staged) is a tiny replacement for `lint-staged`.
+
+Benchmarks show it is much faster, and comes in with a much smaller footprint (currently only 47kB, 1 dependency).
+
+This is another easy migration we will soon be kicking off across the ecosystem :rocket:
+
+### milliparsec
+
+In the web server space, [milliparsec](https://github.com/tinyhttp/milliparsec) has gained a huge amount of usage as a replacement for `body-parser`.
+
+At only 11kB, this is much smaller than older alternatives, and much faster!
+
+Excellent work by [v1rtl](https://github.com/talentlessguy) on building this and actively working on feedback from the community :heart:
+
+## Contributions / Improvements
+
+### Storybook
```

Comment:
Not sure if this is intentional as it doesn't have any content, but otherwise the rest LGTM

Reply:
we're still waiting on the storybook team for a summary of a few changes so I can write this section FYI